academic-services/knowledge-exploration-service/tutorial-grammar-design.md

```diff
@@ -0,0 +1,401 @@
+| multiple attribute search | `papers from microsoft about machine learning` | `<rule name=\"#SearchPapers\">papers from <attr name=\"publications#AuthorAffiliations.NormalizedAffiliationName\">microsoft</attr> about <attr name=\"publications#FieldsOfStudy.NormalizedName\">machine learning</attr><end/></rule>` |
+| multiple attribute search without scope terms |  `microsoft machine learning` | `<rule name="#SearchPapers"><attr name="publications#AuthorAffiliations.NormalizedAffiliationName">microsoft</attr><attr name="publications#FieldsOfStudy.NormalizedName">machine learning</attr><end/></rule>` |
+| keyword search | `microsoft academic applications` | `<rule name=\"#SearchPapers\"><attr name=\"publications#NormalizedTitleWords\">microsoft</attr> <attr name=\"publications#NormalizedTitleWords\">academic</attr> <attr name=\"publications#NormalizedTitleWords\">applications</attr><end/></rule>` |
+| keyword search + drop terms | `microsoft garbageterm garbageterm academic garbageterm applications`| `<rule name=\"#SearchPapers\"><attr name=\"publications#NormalizedTitleWords\">microsoft</attr><rule name=\"#DroppedWord\"> garbageterm</rule><rule name=\"#DroppedWord\"> garbageterm</rule> <attr name=\"publications#NormalizedTitleWords\">academic</attr><rule name=\"#DroppedWord\"> garbageterm</rule> <attr name=\"publications#NormalizedTitleWords\">applications</attr><end/></rule>` |
+| multiple attribute search + drop terms | `microsoft garbageterm machine learning garbageterm`| `<rule name=\"#SearchPapers\"><attr name=\"publications#AuthorAffiliations.NormalizedAffiliationName\">microsoft</attr><rule name=\"#DroppedWord\"> garbageterm</rule> <attr name=\"publications#FieldsOfStudy.NormalizedName\">machine learning</attr> garbageterm<end/></rule>` |
+
+## Create a client application that uses the MAKES API instance
+
+Now that we have deployed a MAKES API instance that uses the custom index and grammar, the last step is to create a frontend client that gives users the ability to browse, search, and filter library publications. In the remaining sections of the tutorial, we will be using the sample UI code to illustrate how to retrieve data and generate filters via MAKES Evaluate and Histogram APIs.
+
+### Explore the library browser application sample
+
+The code for the library browser application sample is part of the standard MAKES deployment. After the MAKES API instance with the custom index build has been fully deployed, you can use the library browser application by visiting the following URL:
+
+`<Makes_Instance_Url>/examples/privateLibraryExample/index.html`
```

Comment:
Same feedback as the previous tutorial, can we *please* also create a .zip file with all of these files and put it in the deployment? Having to download these individually is painful!